src/website/views/components/button.pug

```diff
@@ -184,7 +184,7 @@ h4 Labeled icon buttons
       </div>
     </button>
 
-h4 Floating icon buttons
+h3 Floating icon buttons
```

Comment:
Shouldn't we assign aria-labels for these buttons as well?

Reply:
@janibolkvadze good catch again but let's fix it in another PR

Reply:
@janibolkvadze @jllr Yes, I created a component specific ticket to solve - PE-3843